lib/inspec/profile.rb

```diff
@@ -211,6 +212,44 @@ def locked_dependencies
       @locked_dependencies ||= load_dependencies
     end
 
+    def lockfile_exists?
```

Comment:
I agree we need to increase the separation between Lockfile and Profile. @ksubrama and I discussed that a bit and I think the work to change how profile loading works will drive some of that out.  However, in terms of the lockfile_exists? method specifically, I'm not sure it is true that we don't want profile knowing about lockfiles at all since the lockfile now lives in the profile directory.